wallet/src/wallet/tests.rs

```diff
@@ -31,16 +31,25 @@ use common::{
     },
     primitives::Idable,
 };
-use crypto::key::hdkd::{
-    child_number::ChildNumber, derivable::Derivable, derivation_path::DerivationPath,
+use crypto::{
+    key::hdkd::{child_number::ChildNumber, derivable::Derivable, derivation_path::DerivationPath},
+    random::{CryptoRng, Rng},
 };
+use rstest::rstest;
+use test_utils::random::{make_seedable_rng, Seed};
 use wallet_types::{account_info::DEFAULT_ACCOUNT_INDEX, utxo_types::UtxoType};
 
 // TODO: Many of these tests require randomization...
 
 const MNEMONIC: &str =
     "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about";
 
+const NETWORK_FEE: u128 = 10000;
+
+fn gen_random_password(rng: &mut (impl Rng + CryptoRng)) -> String {
+    (0..rng.gen_range(1..100)).map(|_| rng.gen::<char>()).collect()
```

Comment:
Given that we changed this to 1..100, do we have a test that verifies the behavior of an empty password?

Reply:
added an error for empty passwords and 2 tests in the password and wallet modules